.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
I think we could remove the install parameter as in this phase it would just  "... install any dependencies required ..." [1]  and then in the next phase from travis' lifecycle it would "... run a script that effectively runs the build or the tests ..."

[1] http://docs.travis-ci.com/user/build-lifecycle/